releaseNotes/JavaScriptProvider.md

```diff
@@ -2,6 +2,13 @@
 
 [Source code][source_code] | [Package (npm)][package] | [Samples][samples]
 
+## 1.0.0-preview.3 - March 21, 2023
+## Enhancements
+- Added support for updating configuration on demand without causing an application to restart. Dynamic refresh can be enabled by passing `RefreshOptions` when loading the configuration. A new API, `refresh`, has been added to trigger the refresh operation. When this API is called, it checks the server for any new changes to the `watchedSettings` after a specified time interval, `refreshIntervalInMs`. Once there is a change detected on a watched setting, all loaded settings will be updated. See usage in [this example](https://github.com/Azure/AppConfiguration-JavaScriptProvider/blob/main/examples/refresh.mjs). [#21](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/21)
+- Added support for consuming settings as an object. A new API, `constructConfigurationObject`, has been added to construct an object based on the hierarchical keys of the loaded settings. It helps to minimize code changes when switching from local configuration to Azure App Configuration. [#49](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/49)

```

Comment:
Added support for consuming ~settings~ _configuration_ as an object. A new API, `constructConfigurationObject`, has been added to construct a _configuration_ object based on the ~hierarchical keys of the~ key-values loaded ~settings~ _from Azure App Configuration_. It ~helps to~ minimizes _necessary_ code changes _for applications that were using_ ~when switching from local~ _JSON_ configuration _file_ to _adopt_ Azure App Configuration.

Reply:
updated in c880a61